Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN COMMIT=`git rev-parse HEAD` && echo "export const portalCommit = \"${COMMIT}
     && VERSION=`git describe --always --tags` && echo "export const portalVersion =\"${VERSION}\";" >>src/versions.js \
     && /bin/rm -rf .git \
     && /bin/rm -rf node_modules \
-    && npm config set unsafe-perm=true && npm ci \
+    && npm config set unsafe-perm=true && npm i --force \
```

Comment:
doing a trick here since `npm ci` won't build dependcies on github

will switch back to `npm ci` after releasing guppy